assets/models/system/microsoft-phi-2/model.yaml

```diff
@@ -1,6 +1,6 @@
 path:
   container_name: models
-  container_path: huggingface/microsoft-phi-2/1704358140/mlflow_model_folder
+  container_path: huggingface/microsoft-phi-2/1709113941/mlflow_model_folder
   storage_name: automlcesdkdataresources
   type: azureblob
```

Comment:
SHA in model spec? 
version change

Reply:
added sha in spec.yaml
version is already changed in last PR that version is not released yet